v5/ooapi.v5.core/Services/AcademicSessionsService.cs

```diff
@@ -1,11 +1,12 @@
 ﻿using ooapi.v5.core.Models;
 using ooapi.v5.core.Repositories;
+using ooapi.v5.core.Services.Interfaces;
 using ooapi.v5.core.Utility;
 using ooapi.v5.Models;
 
 namespace ooapi.v5.core.Services
 {
```

Comment:
use file scoped namespaces, just as in other classes.

Reply:
ik zie nog steeds een accolade op regel 8 , en mis een puntkomma achter regel 7 ;-)